### ISSUE TITLE
Workflows: galaxy-importer fails due to missing module community.general.pids

### ISSUE THREAD

**Issue by berndfinger** [sap-linuxlab/community.sap_install#716]
See https://ansible.softwarefactory-project.io/zuul/build/69ccd596b20c4ef5bc0789fcd70f4fc6 or https://ansible.softwarefactory-project.io/zuul/build/69ccd596b20c4ef5bc0789fcd70f4fc6/log/job-output.txt#672 .

The error message is:
`WARNING: roles/sap_swpm/tasks/swpm.yml:74:3: 'syntax-check[unknown-module]:' 'couldn'\''t' resolve module/action ''\''community.general.pids'\''.`

**Comment by marcelmamula** [comment]
@berndfinger Wouldn't this become redundant change when https://github.com/sap-linuxlab/community.sap_install/pull/720 is implemented?

**Comment by berndfinger** [comment]
> @berndfinger Wouldn't this become redundant change when #720 is implemented?

Yes, that was one of the justifications for opening #720. However, as per https://github.com/sap-linuxlab/community.sap_install/issues/709#issuecomment-2066181345, changing essential code in `sap_swpm` will trigger a lot of testing and thus take some time before we can integrate it, so both of #709 and #720 might take some time. Maybe fixing #716 can be done earlier.

**Comment by ja9fuchs** [comment]
@berndfinger Is this still an issue? I thought it was fixed in https://github.com/sap-linuxlab/community.sap_install/pull/786?
